### ISSUE TITLE
Binary release related

### ISSUE THREAD

**Issue by JaehyunShim** [roboticsgroup/roboticsgroup_gazebo_plugins#17]
Hi, 

I would like to ask if you can binary-release this package so users can use the plugins in it without compiling it together every time. 

Ryan

**Comment by costashatz** [comment]
> I would like to ask if you can binary-release this package

No unfortunately there is no binary release at the moment. We should aim at making a release. @nlamprian is it possible for you to make a release for the package as I do not have time at the moment? [Instructions on how to do so](http://wiki.ros.org/ROS/ReleasingAPackage). If not, I can do it but later in the year.

> use the plugins in it without compiling it together every time

You only need to compile it once in every machine.

**Comment by nlamprian** [comment]
Sure thing. I'll take care of it soon

**Comment by costashatz** [comment]
> Sure thing. I'll take care of it soon

Thanks a lot!

**Comment by JaehyunShim** [comment]
@nlamprian @costashatz  

Thanks a lot!!!!

Ryan

**Comment by costashatz** [comment]
@nlamprian any news on this one?

**Comment by nlamprian** [comment]
I'll address this next week. Sorry for the delay. Hectic times.

**Comment by nlamprian** [comment]
We got a complaint about the name of the package. I know we are The robotics group, but it can be confusing for people. Do we rename?

https://github.com/ros/rosdistro/pull/26189#pullrequestreview-468427516

**Comment by nlamprian** [comment]
So, to get this over with, I'll go with the name roboticsgroup_upatras_gazebo_plugins.

To avoid disrupting the work of others, I'll fork the repo under a new name, and deprecate this package.

I hope everyone is pleased with the result.

**Comment by costashatz** [comment]
> roboticsgroup_upatras_gazebo_plugins

I do not really like the name, but I do not have any other good proposal for now.

So if I do not get back to you by tomorrow night, move on with this name.

Thanks a lot for handling this!

**Comment by costashatz** [comment]
@nlamprian it will also be a good idea to update the contributors to include yourself. Maybe we can also adapt the license?

**Comment by nlamprian** [comment]
I took the liberty of adding myself as a maintainer. I'm fine with just this change :)

Regarding the release, I created [roboticsgroup_upatras_gazebo_plugins](https://github.com/roboticsgroup/roboticsgroup_upatras_gazebo_plugins) and released the package for melodic and noetic. Both of them will show up with the next distribution sync. When that happens, I'll commit a deprecation message for roboticsgroup_gazebo_plugins, and close any open issues.
